### ISSUE TITLE
Availability for Python 3.8.?

### ISSUE THREAD

**Issue by justforthefun** [jeroen-dhollander/python-more-or-less#1]
Hey,
is it possible to use this in ongoing Python versions like 3.8. (+ in the Pypi libraries)?
Thanks for this great module!

**Comment by jeroen-dhollander** [comment]
Hey justforthefun,

Thanks for your interest in my little project :)

I see no reason why this would not work on 3.8, but I have not tried it myself.
Is there anything stopping you from using it? Do you see any errors?
If so let me know and I'll see what I can do.

**Comment by justforthefun** [comment]
```
Traceback (most recent call last):
  File "c:/.../xy.py", line 10, in <module>
    import more_or_less as ml
  File "C:\...\Python\Python38-32\lib\site-packages\more_or_less\__init__.py", line 4, in <module>
    from .more_page_builder import MorePageBuilder
  File "C:\...\Python\Python38-32\lib\site-packages\more_or_less\more_page_builder.py", line 6, in <module>
    from .terminal_input import TerminalInput
  File "C:\...\Python\Python38-32\lib\site-packages\more_or_less\terminal_input.py", line 4, in <module>
    import termios
ModuleNotFoundError: No module named 'termios'
```

**Comment by jeroen-dhollander** [comment]
Thanks for the stacktrace.
I think this is not a Python 3.8 issue, but caused by the fact that you are on Windows.
Is that correct?

I'll see if I can get reproduce it and implement a cross platform solution.

**Comment by jeroen-dhollander** [comment]
I just pushed a fix for the import issue. It will allow you to use this library on Windows.

Note that reading from stdin is still not possible on Windows, so you will not be able to run any of the examples. I'll see if I can get that working next.

Let me know if you already want me to publish this in the pip package - otherwise I'll wait for my stdin fix.

**Comment by jeroen-dhollander** [comment]
The examples should now work on Windows - let me know if you see any issues.

**Comment by jeroen-dhollander** [comment]
I released version 1.0.2 on pypi, so you should be able to pip install it.
